### PR TITLE
community/nextcloud-client: re-enable all architectures, modernize

### PR DESCRIPTION
**Pull request by PureTryOut**
CC @rnalrd

### CODE REVIEW

**Comment by Cogitri** [comment]
Not really related to this, but does it work for you now?

**Comment by PureTryOut** [comment]
I'll be honest, I haven't tried running it (I don't have a pure x86 installation anywhere). But the comment said that x86 was disabled due to qt5-qtwebengine not being available (which is not the case anymore), so re-enabling it makes sense. I don't see why it wouldn't work on x86, but I guess someone would have to test.

**Comment by Cogitri** [comment]
Ah, sorry that I wasn't clear, I was just curious if it worked on x86_64 for you, QtWebengine still crashes instantly for me :c 

Some user posted a patch to use Nextcloudclient without QtWebengine (requiring you to use an app password for Nextcloud), so maybe we should add that for now?

**Comment by PureTryOut** [comment]
Ah ok. Yeah crashes instantly for me too, it has more problems anyway. If that patch can make it work right now, I'd go for it. But eventually we should move back to QtWebengine.

**Comment by Cogitri** [comment]
Here's the URL to the patch: http://ix.io/1Jld

**Comment by PureTryOut** [comment]
Thanks, I've applied that patch and removed `qt5-qtwebengine` from the `$makedepends` for now. Because we don't use that dependency anymore, I've also re-enabled the package for all architectures.

From the quick testing I did it seems to work!

**Comment by Cogitri** [comment]
Yup, works for me too. Thanks for checking! :)

File: community/nextcloud-client/APKBUILD [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
 builddir="$srcdir"/desktop-$pkgver
 
 build() {
-	cd "$builddir"
 	cmake \
-		-DCMAKE_BUILD_TYPE="Release" \
+		-DCMAKE_BUILD_TYPE=RelWithDebInfo \
 		-DCMAKE_INSTALL_LIBDIR=lib \
 		-DCMAKE_INSTALL_PREFIX=/usr \
 		-DCMAKE_INSTALL_SYSCONFDIR=/etc/$pkgname
```
Comment (Cogitri) [inline community/nextcloud-client/APKBUILD]: I guess we should add `-DNO_SHIBBOLETH=1` here to get rid of the qt5-qtwebkit dependency, upstream has supported app passwords for some years now (and qt5-qtwebkit is super insecure at this point).

**Comment by maxice8** [comment]
@rnalrd

**Comment by PureTryOut** [comment]
Wait, not the full thing got merged. It still has the webkit dependency and lacks `-DNO_SHIBBOLETH=1`.

**Pull request closed without merging**